ci/sandbox/deploy-pipeline.yaml

```diff
@@ -0,0 +1,165 @@
+---
+apiVersion: concourse.k8s.io/v1beta1
+kind: Pipeline
+metadata:
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: deploy
+spec:
+  exposed: true
+  pipelineString: |
+
+    github_source: &github_source
+      uri: https://github.com/alphagov/verify-metadata-controller.git
+      organization: alphagov
+      owner: alphagov
+      repository: verify-metadata-controller
+      github_api_token: ((github.api-token))
+      access_token: ((github.api-token))
+      approvers: ((trusted-developers.github-accounts))
+      required_approval_count: 2
+      commit_verification_keys: ((trusted-developers.gpg-keys))
+
+    harbor_source: &harbor_source
+      username: ((harbor.harbor_username))
+      password: ((harbor.harbor_password))
+      harbor:
+        url: ((harbor.harbor_url))
+        prevent_vul: "false"
+      notary:
+        url: ((harbor.notary_url))
+        root_key: ((harbor.root_key))
+        delegate_key: ((harbor.ci_key))
+        passphrase:
+          root: ((harbor.notary_root_passphrase))
+          snapshot: ((harbor.notary_snapshot_passphrase))
+          targets: ((harbor.notary_targets_passphrase))
+          delegation: ((harbor.notary_delegation_passphrase))
+
+    task_toolbox: &task_toolbox
+      type: docker-image
+      source:
+        repository: govsvc/task-toolbox
+        tag: 1.5.0
+
+    resource_types:
+
+    - name: github
+      type: registry-image
+      source:
+        repository: govsvc/concourse-github-resource
+        tag: v0.0.1
+
+    - name: harbor
+      type: docker-image
+      privileged: true
+      source:
+        repository: govsvc/gsp-harbor-docker-image-resource
+        tag: 0.0.1553882420
+
+    resources:
+
+    - name: release
+      type: harbor
+      icon: tag
+      source:
+        <<: *harbor_source
+        repository: registry.((cluster.domain))/eidas/vmc-release
+
+    jobs:
+
+    - name: deploy
+      serial: true
+      plan:
+
+      - get: release
+        trigger: true
+
+      - task: extract-release
+        image: release
+        config:
+          platform: linux
+          outputs:
+          - name: release
```

Comment:
Does this output, `release` conflict at all with the image name `release`?

Reply:
Nope, it doesn't (I guess it's just the way Concourse works - and we use this technique in a few places).
Concourse creates a directory called `release` that can be shared between tasks. We don't need to access the `release` image anymore after this task so that's okay.

Reply:
Actually, it just occurred to me that the reason it doesn't clash is because all resources in Concourse are directories with a specific file structure. So the `release` resource of `docker-image` is really just a directory called `release` with a bunch of files/dirs in it and the resource is just an abstraction over that. The output `release` makes the new stuff added to that directory available to other tasks.

Reply:
Yep, that makes sense.